tekton/pipelines/binary-container.yaml

```diff
@@ -287,3 +287,20 @@ spec:
           value: "$(context.pipelineRun.name)"
         - name: user-params
           value: '$(params.user-params)'
+    - name: binary-container-output-dockerfile
+      when:
+        - input: "$(tasks.status)"
+          operator: in
+          values:
+            - Failed
```

Comment:
I'd say this doesn't need to be conditional, the original Dockerfile might be useful for successful builds too

(and if it's not conditional, I wonder if it still makes sense to have it as a separate task)

Reply:
yeah it should be consistent, for both successful & failed builds

and I'd just add it to some exit task, this doesn't even deserve it's own exit plugin (maybe store_metadata)

but also having it as separate task doesn't make sense

Reply:
Have opened another PR #1942. This one will be closed.